patterns/nvidia-gpu-efa/README.md

```diff
@@ -17,46 +17,52 @@ The following components are demonstrated in this pattern:
 
 ## Code
 
-```terraform hl_lines="24-26 32-67"
-{% include  "../../patterns/nvidia-gpu-efa/eks.tf" %}
-```
+The code consists of the following files:
 
-```terraform hl_lines="5-47"
-{% include  "../../patterns/nvidia-gpu-efa/helm.tf" %}
-```
+[eks.tf](eks.tf) - template of the EKS cluster, containing a default node group and nvidia-efa node group.
+
+[helm.tf](helm.tf) - helm charts to install [nvidia-device-plugin](https://github.com/NVIDIA/k8s-device-plugin) and [efa-device-plugin](https://github.com/aws-samples/aws-efa-eks)
+
+[main.tf](main.tf) - main project template, includes [`eks.tf`](eks.tf) and [`helm.tf`](helm.tf)
 
 ## Deploy
 
 See [here](https://aws-ia.github.io/terraform-aws-eks-blueprints/getting-started/#prerequisites) for the prerequisites and steps to deploy this pattern.
 
 ## Validate
 
-!!! note
+Note:
```

Comment:
please revert - this is for rendering correctly on doc site 
<img width="711" alt="image" src="https://github.com/aws-ia/terraform-aws-eks-blueprints/assets/10913471/8e9e7161-6b03-41a3-8002-a59e250bdebc">